pkg/R/RDD.R

```diff
@@ -97,18 +97,18 @@ PipelinedRDD <- function(prev, func) {
 
 
 # The jrdd accessor function.
-setGeneric("getJRDD", function(rdd, ...) { standardGeneric("getJRDD") })
-setMethod("getJRDD", signature(rdd = "RDD"), function(rdd) rdd@jrdd )
-setMethod("getJRDD", signature(rdd = "PipelinedRDD"),
-          function(rdd, dataSerialization = TRUE) {
-            if (!is.null(rdd@env$jrdd_val)) {
-              return(rdd@env$jrdd_val)
+setGeneric("getJRDD", function(x, ...) { standardGeneric("getJRDD") })
```

Comment:
Good call, I didn't think about the inheritance issue.

Just switched it back to `rdd`